frontend_server/lib/server.dart

```diff
@@ -354,6 +353,7 @@ Future<int> starter(
     compiler.acceptLastDelta();
     await compiler.recompileDelta();
     compiler.acceptLastDelta();
+    temp.deleteSync(recursive: true);
```

Comment:
maybe this needs to be in finally block of a try finally - in case if any front-end operations would throw an exception